### DISCUSSION TITLE
Using operator learning if dimension of input function is more than 1

### DISCUSSION THREAD

**Discussion by sibyjackgrove** [lululxvi/deepxde#1381] [Q&A]
The examples given for DeepOnet use data sets with only one input function. What should I modify to use more than one input function? Also is there an example for MIOnet?

**Comment by lululxvi** [comment]
See https://doi.org/10.1137/22M1477751 and https://arxiv.org/abs/2303.04778

**Reply by sibyjackgrove** [reply]
Thanks, I tried this out using the implementation in https://github.com/lu-group/mionet/blob/main/training/MIONet_training.py. However, MIONetCartesianProd can only take two input functions. Is there an implementation for more than 2 input functions?

**Reply by lululxvi** [reply]
You can modify the code. It is straightforward.
